test/libsolidity/SolidityNatspecJSON.cpp

```diff
@@ -485,6 +485,411 @@ BOOST_AUTO_TEST_CASE(event)
 	checkNatspec(sourceCode, "ERC20", userDoc, true);
 }
 
+BOOST_AUTO_TEST_CASE(emit_same_signature_event_library_contract)
```

Comment:
Just for future reference: this is beyond the amount of hand-crafted cpp boost test cases, s.t. I'd have transitioned this stuff to nicer `isoltest` tests. But since we have them now like this, it's fine and we can keep them here. But yeah: general policy: more than one test of a few lines like this we should already think about adding a new `isoltest` test kind - the infrastructure for adding simple ones there is rather convenient these days.